src/main/kotlin/com/meltwater/docker/compose/DockerCompose.kt

```diff
@@ -172,15 +216,28 @@ class DockerCompose private constructor(
     }
 
     private fun exec(command: String): String {
-        return executeCommand("docker-compose --project-name $prefix --file $dockerComposeFilename $command", env, NOOP_CONSUMER)
+        return executeCommand(
+            "$dockerCompose --project-name $prefix --file $dockerComposeFilename $command",
+            env,
+            NOOP_CONSUMER
+        )
     }
 
     private fun exec(command: String, listener: (String) -> Unit): String {
-        return executeCommand("docker-compose --project-name $prefix --file $dockerComposeFilename $command", env, listener)
+        return executeCommand(
+            "$dockerCompose --project-name $prefix --file $dockerComposeFilename $command",
+            env,
+            listener
+        )
     }
 
     private fun execAsync(command: String, stdOut: (String) -> Unit, stdErr: (String) -> Unit): ProcessWrapper {
-        return executeCommandAsync("docker-compose --project-name $prefix --file $dockerComposeFilename $command", env, stdOut, stdErr)
+        return executeCommandAsync(
+            "$dockerCompose --project-name $prefix --file $dockerComposeFilename $command",
+            env,
+            stdOut,
+            stdErr
+        )
```

Comment:
Instead of the hard coded "docker-compose" name, we use the binary we found in the path.